.travis.yml

```diff
@@ -30,26 +30,33 @@ install:
 script:
   - export PYTHONPATH=${PWD%/*}
   - ls $PYTHONPATH
-  - if [[ "${CHECK}" == "pytest"  ]];
-    then pytest;
-    elif [[ "${CHECK}" == "pytest_M2Crypto"  ]];
-    then DIRAC_USE_M2CRYPTO=Yes pytest ;
-    elif [[ "${CHECK}" == "pytest_security"  ]];
-    then pytest Core/Security/test;
-    elif [[ "${CHECK}" == "pytest_M2Crypto_security"  ]];
-    then DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test;
-    elif [[ "${CHECK}" == "docs"  ]];
-    then .travis.d/checkDocs.sh;
-    elif [[ "${CHECK}" == "pylint"  ]];
-    then travis_wait 30 .travis.d/runPylint.sh;
-    elif [[ "${CHECK}" == "pylintPY3K"  ]];
-    then travis_wait 30 .travis.d/runPylint.sh;
-    elif [[ "${CHECK}" == "format" ]] && [[ "${TRAVIS_PULL_REQUEST}" != "false" ]];
-    then git remote add GH https://github.com/DIRACGrid/DIRAC.git; git fetch --no-tags GH ${TRAVIS_BRANCH}; git branch -vv; git diff -U0 GH/${TRAVIS_BRANCH} ':(exclude)tests/formatting/pep8_bad.py' | pycodestyle --diff;
+  - |
+    if [[ "${CHECK}" == "pytest"  ]]; then
+      pytest;
+    elif [[ "${CHECK}" == "pytest_M2Crypto"  ]]; then
+      DIRAC_USE_M2CRYPTO=Yes pytest ;
+    elif [[ "${CHECK}" == "pytest_security"  ]]; then
+      pytest Core/Security/test;
+    elif [[ "${CHECK}" == "pytest_M2Crypto_security"  ]]; then
+      DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test;
+    elif [[ "${CHECK}" == "docs"  ]]; then
+      .travis.d/checkDocs.sh;
+    elif [[ "${CHECK}" == "shellcheck"  ]]; then
+      # TODO This should cover more than just tests/CI
+      find tests/CI -name '*.sh' -print0 | xargs -0 -n1 shellcheck --external-sources
+    elif [[ "${CHECK}" == "pylint"  ]]; then
+      travis_wait 30 .travis.d/runPylint.sh;
+    elif [[ "${CHECK}" == "pylintPY3K"  ]]; then
+      travis_wait 30 .travis.d/runPylint.sh;
+    elif [[ "${CHECK}" == "format" ]] && [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
+      git remote add GH https://github.com/DIRACGrid/DIRAC.git; git fetch --no-tags GH ${TRAVIS_BRANCH}; git branch -vv; git diff -U0 GH/${TRAVIS_BRANCH} ':(exclude)tests/formatting/pep8_bad.py' | pycodestyle --diff;
+    else
```

Comment:
This leads to non-PR test runs failing for the "format" check. As is the case for the current integration tests.

Reply:
Was fixed in #4350.